test/testfiles/xliff20/app1/en-US.xliff

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```

Comment:
How are these files used? I don't see a unit test that tests the merging...

Reply:
Actually, I committed files to see the result by running loctool command..  but I've also added test cases too.